tests/freqtradebot/test_freqtradebot.py

```diff
@@ -5460,7 +5460,7 @@ def test_check_and_call_adjust_trade_position(mocker, default_conf_usdt, fee, ca
     assert freqtrade.strategy.adjust_trade_position.call_count == 1
 
     caplog.clear()
-    freqtrade.strategy.adjust_trade_position = MagicMock(return_value=(-10, 'partial_exit_c'))
```

Comment:
I wonder how this worked ... the stake-amount was never -10 something that was owned ...